glslc/test/option_mfmt.py

```diff
@@ -12,12 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import re
```

Comment:
Shouldn't imports be listed alphabetically?
However, I didn't see that in the Python style guide.